.github/ISSUE_TEMPLATE/1-bug-report.md

```diff
@@ -0,0 +1,23 @@
+---
+name: "\U0001F41B Bug report"
```

Comment:
What are the name and about field? It really hard gets the required info from who open the issue. Why not follow up the template here? 

## Description
Add a brief and meaningful description.

## Expected Behavior
Describe the expected behaviour. 

## Actual Behavior
Describe the current/actual behaviour.

## Environment

* Operating system:
* OpenShift versions:
* Project Versions: 

## Steps to reproduce
Describe all steps and pre-requirements which are required to be performed in order to reproduce this scenario. ( E.g 1. Action, 2. Action ... )